rustls/src/verify.rs

```diff
@@ -533,14 +533,25 @@ impl ClientCertVerifier for AllowAnyAuthenticatedClient {
             .map(|crl| crl as &dyn webpki::CertRevocationList)
             .collect::<Vec<_>>();
 
+        let revocation = if crls.is_empty() {
```

Comment:
Maybe spell this as `(!crls.is_empty()).then(|| ..)` instead?

Reply:
going to leave this as-is, as i think @cpu will be deleting it soon to expose the more flexible CRL APIs